proposals/lip-0063.md

```diff
@@ -412,9 +413,9 @@ def createLegacyReserveAccount():
 
```

Comment:
Regarding the line above, I think the following property is not satisfied for the legacy account:

> The lockedBalances array must be in lexicographic order of module name.

Reply:
Good point, explicitly sorted the array after adding the Legacy locked tokens.